src/Voron/StorageEnvironmentOptions.cs

```diff
@@ -572,6 +572,14 @@ private void AttemptToReuseJournal(VoronPathSetting desiredPath, long desiredSiz
 
                         try
                         {
+                            var journalSize = new FileInfo(filename).Length;
+                            if (journalSize > MaxLogFileSize && desiredSize <= MaxLogFileSize)
```

Comment:
1. There is `File.Exists(filename)` below - should we check it first?
2. You can create `new FileInfo(filename)` instance and use its methods here - `Exists` or `MoveTo`.
3. Can we simplify the condition to `journalSize > MaxLogFileSize && desiredSize > journalSize ` - I think it's easier to read.

Reply:
4. Just to make sure - technically we could just skip adding a journal to `_journalsForReuse` list if it's too large. But I think you didn't do it on purpose because next attempt to create a journal could also request such big size. Is that correct?

Reply:
1. done
2. done
3. its not equal to current condition
4. we want to keep the big journals in case we have big tx in future, as well I wanted to do minimal changes to current journal recycle behavior

Reply:
3. OK. I understand it after your clarification in 4.